frontend/src/components/regions/UserActions/utils.js

```diff
@@ -0,0 +1,61 @@
+export function updateBurnDownSnapshots({ burnDownSnapshots }) {
```

Comment:
I wonder if there is a better name for this function. Always remember that good code makes its intentions clear.

Is this meant to fill in the gaps? Maybe `fillInSnapshotDateGaps` or something like that? 

And then if you are struggling to write self-documenting code, comments can be very helpful